### PR TITLE
Menu standardising

### PR DESCRIPTION
**Pull request by mikeallbutt**
Adjusting the menu items to reflect the way the menu is displaying in core theme.

### CODE REVIEW

**Comment by corsacca** [comment]
@mikeallbutt It currently reflects the groups menu. Can you explain the change some more?

**Comment by mikeallbutt** [comment]
These images show that the menu items are more informative. Instead of just saying `all` they say `all {category}`

Screen Shot 2022-03-29 at 07 29 37
Screen Shot 2022-03-29 at 07 30 05
Screen Shot 2022-03-29 at 07 30 52

File: post-type/module-base.php [hunk 1]
```diff
@@ -851,15 +851,15 @@ public static function dt_user_list_filters( $filters, $post_type ){
...
+                    "label" => _x( "Default Filters", 'List Filters', 'disciple-tools-training' ),
                     "count" => $total_all,
                     "order" => 10
                 ];
                 // add assigned to me filters
                 $filters["filters"][] = [
                     'ID' => 'all',
                     'tab' => 'all',
-                    'name' => _x( "All", 'List Filters', 'disciple-tools-training' ),
+                    'name' => _x( "All Trainings", 'List Filters', 'disciple-tools-training' ),
```
Comment (corsacca) [inline post-type/module-base.php:862]: @mikeallbutt let's copy the All %s format

Comment (mikeallbutt) [reply post-type/module-base.php:862]: Ok sounds good. Do I need to do something different in the code then?

Comment (corsacca) [reply post-type/module-base.php:862]: yes. Copy the structure from https://github.com/DiscipleTools/disciple-tools-theme/pull/1609/commits/aee4d6c78bf176401670c55225447d0e96bc960c

**Pull request merged**